### PR TITLE
updating Gaming For Fedora 41

### PR DESCRIPTION
**Pull request by Nightmare1984**
## Type of Change
- [ ] New feature
- [x] Bug fix
- [ ] Documentation update
- [ ] Refactoring
- [x] Hotfix
- [ ] Security patch
- [ ] UI/UX improvement

## Description
Red Hat Already Has OpenH264 Installed And Enabled When You Update To Fedora 41

- Resolves  https://github.com/ChrisTitusTech/linutil/issues/679

## Testing
Since I Do Not Use Fedora I Tested it In Box buddy Virtual Fedora Terminal

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

**Comment by adamperkowski** [comment]
Resolves #679

**Review by adamperkowski — CHANGES_REQUESTED** [review]
These scripts are not explicitly made for Fedora **41**. What about older versions or other distros?

File: core/tabs/system-setup/gaming-setup.sh [hunk 1]
```diff
@@ -36,9 +36,15 @@ installDepend() {
...
             ;;
         dnf)
-            "$ESCALATION_TOOL" "$PACKAGER" install https://mirrors.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm https://mirrors.rpmfusion.org/nonfree/fedora/rpmfusion-nonfree-release-$(rpm -E %fedora).noarch.rpm -y
-            "$ESCALATION_TOOL" "$PACKAGER" config-manager --enable fedora-cisco-openh264 -y
-            "$ESCALATION_TOOL" "$PACKAGER" install -y "$DEPENDENCIES"
+            if [ "$(rpm -E %fedora)" -le 41 ]; then 
+                "$ESCALATION_TOOL" "$PACKAGER" install ffmpeg ffmpeg-libs -y
+                "$ESCALATION_TOOL" "$PACKAGER" install -y "$DEPENDENCIES"
+            else
+                echo "Skipping Fedora 41 Setup To Older setup"
```
Comment (adamperkowski) [inline core/tabs/system-setup/gaming-setup.sh:43]:
```suggestion
                printf "%b\n" "${CYAN}Fedora < 41 detected. Installing rpmfusion repos."
```

**Review by adamperkowski — APPROVED** [review]

File: core/tabs/system-setup/gaming-setup.sh [hunk 2]
```diff
@@ -36,9 +36,15 @@ installDepend() {
...
         dnf)
-            "$ESCALATION_TOOL" "$PACKAGER" install https://mirrors.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm https://mirrors.rpmfusion.org/nonfree/fedora/rpmfusion-nonfree-release-$(rpm -E %fedora).noarch.rpm -y
-            "$ESCALATION_TOOL" "$PACKAGER" config-manager --enable fedora-cisco-openh264 -y
-            "$ESCALATION_TOOL" "$PACKAGER" install -y "$DEPENDENCIES"
+            if [ "$(rpm -E %fedora)" -le 41 ]; then 
+                "$ESCALATION_TOOL" "$PACKAGER" install ffmpeg ffmpeg-libs -y
+                "$ESCALATION_TOOL" "$PACKAGER" install -y "$DEPENDENCIES"
+            else
+                printf "%b\n" "${CYAN}Fedora < 41 detected. Installing rpmfusion repos.${RC}"
+                "$ESCALATION_TOOL" "$PACKAGER" install https://mirrors.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm https://mirrors.rpmfusion.org/nonfree/fedora/rpmfusion-nonfree-release-$(rpm -E %fedora).noarch.rpm -y
```
Comment (nnyyxxxx) [inline core/tabs/system-setup/gaming-setup.sh:44]:
```suggestion
                "$ESCALATION_TOOL" "$PACKAGER" install https://mirrors.rpmfusion.org/free/fedora/rpmfusion-free-release-"$(rpm -E %fedora)".noarch.rpm https://mirrors.rpmfusion.org/nonfree/fedora/rpmfusion-nonfree-release-"$(rpm -E %fedora)".noarch.rpm -y
```

**Comment by nnyyxxxx** [comment]
put

- resolves #679

in the description

**Review by nnyyxxxx — APPROVED** [review]

**Review by ChrisTitusTech — APPROVED** [review]

**Pull request merged**